dix/req.c

```diff
@@ -1625,24 +1628,31 @@ static void dix_rop_completed(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 	struct m0_dix_rop_ctx *rop_del_phase2 = NULL;
 	bool                   del_phase2 = false;
 	struct m0_dix_cas_rop *cas_rop;
+	int                    successful_ops = 0;
 
 	(void)grp;
 	if (req->dr_type == DIX_NEXT)
 		m0_dix_next_result_prepare(req);
 	else {
+		m0_tl_for (cas_rop, &rop->dg_cas_reqs, cas_rop) {
+			if (cas_rop->crp_creq.ccr_sm.sm_rc == 0) {
+				successful_ops++;
+			}
+		} m0_tl_endfor;
```

Comment:
Something like
```
	successful_ops = m0_tl_reduce(cas_rop, scan, &rop->dg_cas_reqs, 0,
				  + !!(cas_rop->crp_creq.ccr_sm.sm_rc == 0));
```
might be more idiomatic.

Reply:
Changed it